pkg/controller/provider/web/ocp/client.go

```diff
@@ -158,8 +158,13 @@ func (r *Finder) ByRef(resource interface{}, ref base.Ref) (err error) {
 		}
 		name := ref.Name
 		if name != "" {
-			ns, name := path.Split(name)
-			ns = strings.TrimRight(ns, "/")
+			var ns string
+			if ref.Namespace != "" {
```

Comment:
worth documenting when we expect the namespace to be missing?

Reply:
yes, will add